README.md

```diff
@@ -13,10 +13,20 @@ This is the website for the Free and Open Source Software initiative at the [RIT
 This website includes content for community initiatives, FOSS academic courses, and more.
 It is also used to communicate announcements to the RIT FOSS community.
 
+This repository is how to edit the website [Fossrit.Github.io](https://fossrit.github.io/).
+
 
 ## How to contribute
 
-See the [_Website (fossrit.github.io)_](https://runbook.fossrit.community/infra/website/ "Website (fossrit.github.io) - FOSSRIT Runbook") page in the [FOSSRIT Runbook](https://runbook.fossrit.community/).
+1. Fork this repository!
+2. Make your desired changes to either add additional features or repair existing ones.
+3. Commit your changes to your forked repository.
+4. Push to your forked repository's main branch.
```

Comment:
This generally isnt the recommended strategy. I would either say nothing or encourage people to make feature branches

Reply:
I agree. Changes will be made to reflect that strategy.